### PR TITLE
Update the website to reflect that 1.6 is the new LTS

### PR DESCRIPTION
**Pull request by ViralBShah**
Most likely this can be merged when 1.7 is released.

### CODE REVIEW

**Review by oscardssmith — APPROVED** [review]

**Review by StefanKarpinski — APPROVED** [review]

**Review by ChrisRackauckas — APPROVED** [review]

**Review by vtjnash — APPROVED** [review]
Is this the only place we need to update it? It seems almost too simple. Is there any CI script updates too, or press releases?

**Review by sara192 — APPROVED** [review]

File: config.md [hunk 1]
```diff
@@ -13,9 +13,9 @@
 @def stable_release = "1.6.2"
 @def stable_release_short = "1.6"
 @def stable_release_date = "July 14, 2021"
-@def lts_release = "1.0.5"
-@def lts_release_short = "1.0"
-@def lts_release_date = "Sep 9, 2019"
+@def lts_release = "1.6.2"
```
Comment (oxinabox) [inline config.md:16]:
```suggestion
@def lts_release = "1.6.3"
```
right?

Comment (DilumAluthge) [reply config.md:16]:
```suggestion
@def lts_release = "1.6.3"
```

**Comment by oscardssmith** [comment]
Time to merge?

**Comment by ararslan** [comment]
Closed by #1441

**Pull request closed without merging**